tc/sgx/common/packages/db_store/lmdb_store.cpp

```diff
@@ -466,4 +466,4 @@ std::string tcf::db_store::db_store_get_all(
 #endif
 
     return table_keys;
-}
+}
```

Comment:
Please add new line at end of file

Reply:
done as e91d6a89dd498d293d8295b983d4c0612af5a5ce

Reply:
done as e91d6a89dd498d293d8295b983d4c0612af5a5ce